src/screens/atomic-swap/BalanceComponent.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+import helpers from '../../utils/helpers'
+import { t } from "ttag";
+
+/**
+ * @param {PartialTx} partialTx
+ * @param {HathorWallet} wallet
+ * @param {DisplayBalance[]} balance
+ */
+export function BalanceComponent ({ partialTx, wallet, balance }) {
```

Comment:
I still think we should have something in the wallet balance component warning the user that there is an amount of tokens that are locked into atomic swap proposals.

Reply:
This feature will be implemented when the integration with the backend is complete, since we'll be able to know which balances are locked at wallet load time.

For now, the only time when these balances are updated is when the user opens the `Atomic Swap` screen.

Reply:
Do you have a task in the project to do it?

Reply:
Yes, a task was added to ensure this will be done